src/main/java/uk/gov/hmcts/reform/roleassignment/util/ValidationUtil.java

```diff
@@ -152,7 +152,7 @@ public static void validateRequestedRoles(Collection<RoleAssignment> requestedRo
             }
 
             validateId(Constants.NUMBER_TEXT_HYPHEN_PATTERN, requestedRole.getActorId());
-            compareRoleType(requestedRole.getRoleType().toString());
+            //compareRoleType(requestedRole.getRoleType().toString());
```

Comment:
Do we need this piece of code? If it is not needed we can remove the line. But I think there is a use for it right..?

Reply:
No we don't need it.  It was commented out to be removed.  I've now removed it entirely